Prev next links

davidfarmer opened the pull request:
The Prev/UP/Next links now function  properly, and the top level is numbered in the TOC.

Note that the "level" in the TOC is one larger than I expected.  Will submit an issue.

siefkenj commented:
I realized that this is being done the wrong way...what should be done is setting the
```
as={InternalAnchor}
```
attribute on each `ToolbarItem`. Then `InternalAnchor` will handle suppressing the default event. Centralizing everything to `InternalAnchor` will allow us to fix the "open in a new tab" issue all at once.

Also the `href` is wrong, as a `#` is needed before the page id.

davidfarmer commented:
I changed "as" to InternalAnchor, and removed the now un-necessary e.preventDefault .
Separate commits in case I misunderstood.

I am not sure what to do with the comment about needing `#` before the page id

siefkenj commented:
@davidfarmer The `onClick` handler should be completely removed because clicks are handled by `InternalAnchor` already. As for the `#` it should be something like

```
href={"#" + (navTargets.prev?.id || "")}
```

so that the href always starts with `#`

siefkenj commented:
You might have been confused because I was giving you the wrong instructions...What actually should have been done is `navTarges.pref?.href` should have been used!

I've fixed this in PR #13 

I've also included all the other changes from this PR, so I'm closing for now.

siefkenj commented:
@davidfarmer PS: #13 also updates to the newest recompiled article, so you'll need to do a `git submodule update` when you pull from master next.
